Stream title & game in live notifcations (Twitch) #111

MadMakz opened the pull request:
![red-discordbot-i111-p198](https://cloud.githubusercontent.com/assets/3930214/14833214/6b6575e0-0bfe-11e6-977f-93b1daa2abe0.png)

Commit:
Stream title & game in live notifcations (Twitch) #111

tekulvw commented on cogs/streams.py:
```diff
@@ -39,9 +39,8 @@ def __init__(self, bot):
-        if online is True:
-            await self.bot.say("http://www.twitch.tv/{} "
-                               "is online!".format(stream))
+        if online is not False and online is not "error":
```
`if online and online != "error":`
^ that is what this line should read, same with line 313

tekulvw commented:
Also, we've set up a new way to contribute. Any changes you'd like to make should now be PR'd to the `cog-changes` branch.

MadMakz committed:
fetch master (#1)

* Owner cog / Version command / doc link update / OAUTH (#203)

* Version command (#183)

* reword. replace link with link to our doc

* Move all owner commands to a separate plugin (#188)

* Move all owner commands to their own plugin.

* Move all initial cog loading to Owner plugin

* Final fix of initial cog loading

* don't allow people to unload the owner plugin without reloading it

* make sure we modify the cog registry like we're supposed to

* log the functionname too

Message updates, grammar, politeness etc

Get right names in logging formatter

Ignore cogs.owner if we find it

* add version back in

add reload docstring

Heh, woops...little security bug here

* Add in globals and bot to locals

* pass exception to the logger

* Bot will now generate OAUTH URL from the supplied endpoint (#196)

* Formatting changes, revert uptime

* Store the oauth_url internally, provide it if the owner uses `!join`

* Fix filters audio.py, added avconv toggle (#210)

!audioset player

* Don't try to interpret clientexceptions on cog load

* Fixed initial cogs loading

* Added !set nickname and !rename <user>

* Quick and dirty fix for !cleanup

* Added way to reset nicknames

* Added !servers

Lists and allows to leave servers from anywhere

* Hotfix to downloader

So we stop nuking data folders...sorry for past user settings :/

MadMakz commented:
obviusly i'm too stupid for gits pull/push so i'm closeing this
